src/components/Markdown.react.js

```diff
@@ -66,6 +106,11 @@ DashMarkdown.propTypes = {
         PropTypes.arrayOf(PropTypes.string),
     ]),
 
+    /**
+     * Config options for syntax highlighting.
+     */
+    highlight_config: PropTypes.object,
```

Comment:
This could be more specific — so far it accepts just one parameter.

Reply:
Good call to make this an object we can extend later, but yeah, let's make it `exact` and enumerate its contents. Also:
- rather than `dark: false | true` let's do `theme: 'light' | 'dark'` so we can add more themes later if we choose.
- give it a `defaultProps` entry of `{}`, that'll simplify the logic a bit. (You're probably aware of this, but don't put nested values like `{theme: 'light'}` in a default, that leads to unpleasant consequences when the user overrides part but not all of the prop)